pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md

```diff
@@ -91,16 +91,43 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
 You must create a Kubernetes `Secret` with AWS access credentials for the backup and then update the Kaptain configuration to enable backup and configure the storage location.
 
 Create a Kubernetes `Secret` in the same namespace where Kaptain is installed using the AWS credentials as follows:
-```yaml
+```bash
+AWS_ACCESS_KEY_ID=<aws_access_key_id>
```

Comment:
```suggestion
AWS_ACCESS_KEY_ID="<aws_access_key_id>"
AWS_SECRET_ACCESS_KEY="<aws_secret_access_key>"
WORKSPACE_NAMESPACE="<workspace_namespace>"
```

Reply:
If you need quotes around your AWS access key id, you have done something wrong. These are always alphanumeric.